twilio/modules/listener/Module.md

```diff
@@ -149,140 +41,61 @@ Callback URL registration method depends on the event type.
-configurable int & readonly port = ?;
-
-listener twilioListener:Listener tListener = new (port, twilioAuthToken, callbackUrl);
-
-service / on tListener {
-    remote function onSmsDelivered(twilioListener:SmsStatusChangeEvent event) returns error? {
-        log:printInfo("Delivered", event);
-    }
-}
+    import ballerinax/twilio.'listener as twilioListener;
 ```
-# Samples
-## Twilio Listener Operations
-
-### QUEUED/SENT/DELIVERED SMS Event
-This examples shows how you can start a ballerina twilio listener using localhost. you will need to use ngork to expose a web server running on your local machine to the internet. Find more sample from here. [Find more samples from here](https://github.com/ballerina-platform/module-ballerinax-twilio/tree/master/samples/listener%20samples)
-
+### Step 2 - Create a new listener instance
 ```ballerina
-import ballerina/log;
-import ballerinax/twilio;
-import ballerinax/twilio.'listener as twilioListener;
-
-configurable string & readonly twilioAccountSid = ?;
-configurable string & readonly twilioAuthToken = ?;
-configurable string & readonly fromNumber = ?;
-configurable string & readonly toNumber = ?;
-configurable string & readonly test_message = ?;
-configurable string & readonly twimlUrl = ?;
-configurable string & readonly callbackUrl = ?;
-configurable int & readonly port = ?;
+    configurable string & readonly twilioAuthToken = ?;
+    configurable string & readonly callbackUrl = ?;
+    configurable int & readonly port = ?;
 
-listener twilioListener:Listener tListener = new (port, twilioAuthToken, callbackUrl);
-
-service / on tListener {
-    remote function onSmsQueued(twilioListener:SmsStatusChangeEvent event) returns error? {
-        log:printInfo("Queued", event);
-    }
-    remote function onSmsSent(twilioListener:SmsStatusChangeEvent event) returns error? {
-        log:printInfo("Sent", event);
-    }
-    remote function onSmsDelivered(twilioListener:SmsStatusChangeEvent event) returns error? {
-        log:printInfo("Delivered", event);
+    listener twilioListener:Listener tListener = new (port, twilioAuthToken, callbackUrl);
+```
+### Step 3 - Add a service 
+1. Add a service with necessary remote functions as below
+```ballerina
+    service / on tListener {
+        remote function onSmsDelivered(twilioListener:SmsStatusChangeEvent event) returns error? {
+            log:printInfo("Delivered", event);
+        }
     }
-}
+```
+2. Use `bal run` command to compile and run the Ballerina program.
 
-public function main() {
-    twilio:TwilioConfiguration twilioConfig = {
-        accountSId: twilioAccountSid,
-        authToken: twilioAuthToken
-    };
-    twilio:Client twilioClient = new (twilioConfig);
-    var details = twilioClient->sendSms(fromNumber, toNumber, test_message, callbackUrl);
-    if (details is error) {
-        log:printInfo(details.message());
+## Quick reference
+Code snippets of some frequently used functions: 
+* QUEUED/SENT/DELIVERED SMS Event
```

Comment:
Have the bullet point header in title case "QUEUED/SENT/DELIVERED SMS event"

Is there a reason for the statuses to be in all caps? You have used title caps in the next bullet point header.